packages/salesforcedx-vscode-apex/package.nls.json

```diff
@@ -20,5 +20,6 @@
   "force_apex_test_run_text": "SFDX: Run Apex Tests",
   "force_apex_test_suite_run_text": "SFDX: Run Apex Test Suite",
   "force_apex_test_suite_create_text": "SFDX: Create Apex Test Suite",
-  "force_apex_test_suite_build_text": "SFDX: Add Tests to Apex Test Suite"
+  "force_apex_test_suite_build_text": "SFDX: Add Tests to Apex Test Suite",
+  "apex_code_completion_stats_description": "Allow Apex Language Server to collect telemetry on code completion usage"
```

Comment:
@sbudhirajadoc This setting allows the user to indicate whether they'd like to report telemetry data with their code completion usage - let me know if you have any feedback!

Reply:
lgtm. thanks!